syft/federated/plan.py

```diff
@@ -458,7 +457,7 @@ def _send(self, location: "sy.workers.BaseWorker"):
         _ = self.owner.send(self, workers=location)
 
         # Deep copy the plan without using deep copy
-        pointer = sy.serde._detail_plan(self.owner, sy.serde._simplify_plan(self))
+        pointer = sy.torch_serde._detail_plan(self.owner, sy.torch_serde._simplify_plan(self))
```

Comment:
Maybe this is a detail, but is there a way to make it transparent? I mean: I'd like to still call sy.serde.detail_<mytype> no matter if detail_<mytype>  is in serde/torch_serde or serde/syft_serde etc
It would require to change the way the imports are done. If this is not too complicated I think it would make it easier to use in non-standard cases like this one.

Reply:
I can do it if change the detail method from protected to public. (from _detail_plan to detail_plan)
Is that ok?

Reply:
Are you forced to do this? I believe it works also with the "_" but maybe I'm wrong

Reply:
You are right. At least after moving _detail_plan in Plan class, it does work

Reply:
Yes in practice python doesn't really care about "protected", it's more like a convention for users and developers

Reply:
To conclude on that, I believe the following happened
serde/init.py :
```
from syft.serde.serde import *
from syft.serde.torch_serde import *
```
In this case it seems that it does not import methods with "_" , 
so I couldn't use sy.serde._simplify() in plan.py for example

So I added in the serde.init.py
```
from syft.serde.serde import _simplify
```
Then it does work

Reply:
Ah ok, good to know! Thanks